settings.php

```diff
@@ -48,6 +48,26 @@
     );
     $settings->add(new admin_setting_configselect($name, $title, $description, $default, $choices));
 
+    /* Toggle display block choices */
+    $name = 'format_topcoll/defaultdisplayblocks';
+    $title = get_string('defaultdisplayblocks', 'format_topcoll');
+    $description = get_string('defaultdisplayblocks_desc', 'format_topcoll');
+    $default = array('0,1,2,3');
+    $choices = array('search_forums','news_items','calendar_upcoming','recent_activity');
+    $settings->add(new admin_setting_configmultiselect($name, $title, $description, $default, $choices));
+    
+    /*  Toggle blocks location. */
```

Comment:
Please use single inline comments for one line '//' or only one block comment '/* .... */'